core/autoscaling.go

```diff
@@ -917,6 +917,9 @@ func (a *autoScalingGroup) detachAndTerminateOnDemandInstance(
 		return err
 	}
 
+	// Wait till detachment initialize is complete before terminate instance
+	time.Sleep(20 * time.Second)
```

Comment:
Please also use the  SleepMultiplier so that we don't actually sleep when running the tests.
This would become`time.Sleep(20 * time.Second * s.region.conf.SleepMultiplier)`

Reply:
I did a typo: should likely be `a.region.conf.SleepMultiplier`

Reply:
Fixed